filebeat/beater/filebeat.go

```diff
@@ -75,7 +75,10 @@ func New(b *beat.Beat, rawConfig *common.Config) (beat.Beater, error) {
 func (fb *Filebeat) modulesSetup(b *beat.Beat) error {
 	esConfig := b.Config.Output["elasticsearch"]
 	if esConfig == nil || !esConfig.Enabled() {
-		return fmt.Errorf("Filebeat modules configured but the Elasticsearch output is not configured/enabled")
```

Comment:
LGTM (couple of minor changes)

Filebeat modules are enabled, but the Elasticsearch output is not configured/enabled. This means Filebeat can't load the Ingest Node pipelines. If you have already loaded the pipelines, you can ignore this warning.

Not sure if this covers LS users, tho, because they really don't need to load the pipeline at all, right?